projects/plutus/inc/ltc_adc.h

```diff
@@ -56,3 +56,6 @@ StatusCode ltc_adc_init(LtcAdcStorage *storage, const LtcAdcSettings *settings);
 // Register a callback to be run whenever there is new data
 StatusCode ltc_adc_register_callback(LtcAdcStorage *storage, LtcAdcCallback callback,
                                      void *context);
+
+// Exposed as a test function
+StatusCode ltc_adc_set_input_voltage(int32_t input_voltage);
```

Comment:
This shouldn't be defined in the public header. You should also prefix this with `test_` or `_test` to make it very obvious it's a test function.

Reply:
Done